.github/workflows/patch-release-version-bump.yml

```diff
@@ -5,40 +5,15 @@ on:
       MAJOR_VERSION:
         type: choice
         description: 'Major version (current)'
-        options:
-        - '7'
-        - '8'
+        options: ['7', '8']
       MINOR_VERSION:
         type: choice
         description: 'Minor version (current)'
-        options: 
-        - '12'
-        - '13'
-        - '14'
-        - '15'
-        - '16'
-        - '17'
-        - '18'
-        - '19'
-        - '20'
-        - '21'
-        - '22'
+        options: ['12', '13', '14', '15', '16', '17', '18', '19', '20', '21', '22', '23', '24', '25']
       PATCH_VERSION:
         type: choice
         description: 'Patch version (new)'
-        options: 
-        - '0'
-        - '1'
-        - '2'
-        - '3'
-        - '4'
-        - '5'
-        - '6'
-        - '18'
-        - '19'
-        - '20'
-        - '21'
-        - '22'
+        options: ['0', '1', '2', '3', '4', '5', '6', '7', '8', '9', '10', '11', '12', '13', '14', '15', '16', '17', '18', '19', '20', '21', '22', '23', '24', '25']
```

Comment:
Let's remove 0 so it isn't confused for a minor bump!

Reply:
```suggestion
        options: ['1', '2', '3', '4', '5', '6', '7', '8', '9', '10', '11', '12', '13', '14', '15', '16', '17', '18', '19', '20', '21', '22', '23', '24', '25']
```

Reply:
Beautiful